client/src/components/common/SprintModal/CreateSprintModal/helpers.ts

```diff
@@ -0,0 +1,10 @@
+export const getNextDate = (duration: number | 'custom', startDate: Date) => {
+	const nextDate = new Date(
+		startDate.getFullYear(),
+		startDate.getMonth(),
+		startDate.getDate() + 7 * (duration === 'custom' ? 1 : duration),
+		startDate.getHours(),
+		startDate.getMinutes() + 15,
+	);
+	return nextDate;
+};
```

Comment:
What if startDate is not Date? Let's handle this behavior too and avoid crashing. [Link](https://trello.com/c/jZYlZHjG/263-add-start-and-end-date-for-the-sprint#action-5f48adb8e6dffc0efb3de3de).